rust/main/src/ast/mod.rs

```diff
@@ -16,10 +16,10 @@ pub enum ElementClosingTag {
 
 #[derive(Copy, Clone, Eq, PartialEq, Debug)]
 pub enum ScriptOrStyleLang {
-    CSS,
```

Comment:
Clippy might warn about this, but I think CSS is the appropriate name to use here. Likewise for JS. Almost certainly it's not worth changing the API to make clippy happy about the names.

Reply:
Personally I'm more on the clippy side of not capitalizing these but it's all very opinionated after all.

Completely agree that it's not worth it from the API perspective.

I'll change these back to the original version and add a linting exception to make clippy happy.